src/ConnectedMode/UI/ManageBinding/ManageBindingViewModel.cs

```diff
@@ -165,8 +166,7 @@ public async Task UnbindWithProgressAsync()
     {
         var bind = new TaskToPerformParams<AdapterResponse>(UnbindAsync, UiResources.UnbindingInProgressText, UiResources.UnbindingFailedText)
         {
-            AfterSuccess = AfterUnbind,
-            AfterProgressUpdated = OnProgressUpdated
+            AfterSuccess = AfterUnbind, AfterProgressUpdated = OnProgressUpdated
```

Comment:
Instead of calling the `AfterUnbind` here and in the `DisplayBindStatusAsync`, it could follow the same logic as the `BindAsync` and call directly the `DisplayBindStatusAsync` to update the UI based on the binding status. 

The reason behind this comment is that all of the UI updates logic is enclosed in the `DisplayBindStatusAsync`, the suggested change will keep the UI updates consistent and in a single place.